### PR TITLE
[WFLY-5651] fix validation errors in a new jboss-web_12_0.xsd

### PR DESCRIPTION
**Pull request by soul2zimate**
fix the validation errors reported in https://issues.jboss.org/browse/WFLY-5651

### CODE REVIEW

**Comment by soul2zimate** [comment]
diff with jboss-web_10_0.xsd:

75c75,76
<    <xsd:import namespace="http://java.sun.com/xml/ns/javaee" schemaLocation="web-app_3_1.xsd"/>
---
>    <xsd:import namespace="http://xmlns.jcp.org/xml/ns/javaee" schemaLocation="http://www.jboss.org/schema/jbossas/web-app_3_1.xsd"/>
>    <xsd:include schemaLocation="http://www.jboss.org/schema/jbossas/jboss-common_6_0.xsd"/>
308c309
<              Thesymbolic-linked-allowed element specifies if symlinking is allowed in exploded deployments.
---
>              The symbolic-linked-allowed element specifies if symlinking is allowed in exploded deployments.
312c313
<           <xsd:restriction base="xsd:boolean"/>
---
>           <xsd:restriction base="javaee:generic-booleanType"/>

**Comment by wolfc** [comment]
I would like to see a test case that validates this xsd.

File: web/src/main/resources/schema/jboss-web_10_1.xsd [hunk 1]
```diff
@@ -0,0 +1,599 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ JBoss, Home of Professional Open Source.
+  ~ Copyright 2012, Red Hat, Inc., and individual contributors
```
Comment (rhusar) [inline web/src/main/resources/schema/jboss-web_10_1.xsd]: Looking at my wristwatch its 2016 already ;-)

Comment (rhusar) [inline web/src/main/resources/schema/jboss-web_10_1.xsd]: Note that this is the file that people will actually be looking at, wrong year here might give them an impression its honestly old, so its a UX thing to get this right.

**Comment by ehsavoie** [comment]
@soul2zimate could you please rebase ?

**Comment by soul2zimate** [comment]
sure, it's been here for quite a while, I will check this.

**Comment by soul2zimate** [comment]
The previous old one is for 10.1. I updated and bumped up version to 12.0 and add one more commit to fix validation type error on flushOnSessionInvalidation in jboss web xsd.

new diff https://gist.github.com/soul2zimate/563b4974bf5c460ccfb6e43fa2714939

**Pull request merged**